File: concrete/src/main/java/pcb/uwu/unit/base/RadianUnit.java
package pcb.uwu.unit.base;

public class RadianUnit extends AngleUnit {

	public static final RadianUnit RADIAN = new RadianUnit();

	public RadianUnit() {
		super();
	}

	@Override
	public String getSymbol() {
		return "rad";
	}

	@Override
	public String getSingularName() {
		return "radian";
	}

	@Override
	public String getPluralName() {
		return "radians";
	}
}

File: concrete/src/main/java/pcb/uwu/unit/base/RadianUnit.kt
package pcb.uwu.unit.base

import pcb.uwu.utils.identity

object RadianUnit
    : AngleUnit(symbol = "rad",
                singularName = "Radian",
                pluralName = "Radians",
                translationToCanonical = identity(),
                translationFromCanonical = identity())
